src/Unleash/Communication/UnleashApiClient.cs

```diff
@@ -21,7 +24,26 @@ internal class UnleashApiClient : IUnleashApiClient
         private readonly UnleashApiClientRequestHeaders clientRequestHeaders;
         private readonly EventCallbackConfig eventConfig;
         private readonly string projectId;
-
+        private int featureClientToSkip = 0;
```

Comment:
Name is a bit confusing to me. `featureRequestsToSkip` maybe?

Reply:
Good point, yes will do! thanks